src/main/java/org/b612foundation/adam/datamodel/PropagatorConfiguration.java

```diff
@@ -0,0 +1,237 @@
+package org.b612foundation.adam.datamodel;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Configuration for the interplanetary orbit propagator. Lists various forces to be included.
+ */
+public class PropagatorConfiguration {
+
+  /**
+   * How to include gravity of individual bodies: ignore it all together, use it as point mass, or use a detailed model.
+   * For now assume that there is only one "good" spherical harmonics model for each body, so we don't specify which one
+   * to use.
+   */
+  public enum PlanetGravityMode {
+    /** Ignore this body's gravity completely. */
+    OMIT,
+    /** Treat the body as point-mass. */
+    POINT_MASS,
+    /** Use a spherical harmonics model. Assumes there is a default model. Not always applicable. */
+    SPHERICAL_HARMONICS
+  }
+
+  /** Globally unique id of this config. */
+  private String uuid;
+  /** Human-readable description. Not used in hash code or equal. */
```

Comment:
It does seem to be used in hash code and equal. Not sure what behavior should be.

Reply:
Hmmm, not sure when it snuck in. Removed from `equals` and `hashCode`